src/main/resources/openapi/notification.yaml

```diff
@@ -0,0 +1,54 @@
+openapi: 3.0.1
+
+info:
+
+  version: 0.0.1
+  title: Modernised DARTS
+  description: |-
+    Modernised DARTS (Digital Audio Recording and Transcription Service).
+
+servers:
+  - url: http://localhost:4550/
+
+
+paths:
+  /notification/create:
```

Comment:
I'm not sure there is a need for an endpoint for creating notifications. I was imaging this would be incorporated within a different endpoint to carry out the whole action, for example `/cases/XXXXX/approve-transcription` (totally made this up).

If there's a good reason, or some other thoughts we can discuss?

Reply:
I thought the agreed solution was the one proposed by Chris B in the KT 2 weeks ago where he mentioned that any module that was calling another module, should do it through an api/controller layer, unless i misunderstood?

Reply:
I might be misunderstood but I didn't think that meant exposing it as an endpoint, but creating an interface under an api package, and an implementation where we delegate the call down. @chrisbellingham-hmcts can you confirm?

Reply:
I share the same interpretation as @davet1985, the api layer would not expose a HTTP endpoint, it's implementation would be a `@Component` or `@Service`.

Reply:
ok, have removed.